components/Ink/src/private/MDCInkLayer.m

```diff
@@ -158,7 +159,7 @@ - (void)startInkAtPoint:(CGPoint)point animated:(BOOL)animated {
     _startAnimationActive = YES;
 
     CAMediaTimingFunction *materialTimingFunction =
-        [[CAMediaTimingFunction alloc] initWithControlPoints:0.4f:0:0.2f:1];
+        [[CAMediaTimingFunction alloc] initWithControlPoints:(float) 0.4:0:(float)0.2:1];
```

Comment:
nit: remove extra spacing

Reply:
We can do a follow-up to disable clang on these lines if it's a big problem.